api/v1/cluster_types.go

```diff
@@ -679,7 +686,7 @@ type ClusterStatus struct {
 
 	// TablespaceStatus reports the state of the declarative tablespaces in the cluster
 	// +optional
-	TablespaceStatus TablespacesState `json:"tablespacesStatus,omitempty"`
+	TablespaceStatus []TablespaceState `json:"tablespaceStatus,omitempty"`
```

Comment:
Going to rename this field to `TablespacesStatus`. We're already using `TablespaceSatus` as an enum pending/reconciled.

Reply:
Don't love it but I think it helps clarify